db/migrate/20170109221226_create_configuration_sources.rb

```diff
@@ -0,0 +1,17 @@
+class CreateConfigurationProjects < ActiveRecord::Migration[5.0]
+  def change
+    create_table :configuration_sources do |t|
+      t.belongs_to  :configuration_manager, :type => :bigint
+      t.string      :manager_ref
+      t.string      :name
+      t.string      :description
+      t.string      :scm_type
+      t.bigint      :scm_reference_id
+      t.boolean     :scm_clean
+      t.boolean     :scm_delete_on_update
+      t.boolean     :scm_update_on_launch
```

Comment:
<img width="1128" alt="screen shot 2017-01-09 at 6 19 08 pm" src="https://cloud.githubusercontent.com/assets/2421248/21821377/52c3269a-d741-11e6-8a48-d86deab2a045.png">

So, we don't need to specify the SCM options? (Or we don't want to support that in this phase?)